include/ignition/physics/ForwardStep.hh

```diff
@@ -49,6 +49,12 @@ namespace ignition
       std::string annotation;
     };
 
+    struct ChangedWorldPoses
+    {
+      std::vector<WorldPose> entries;
+      std::string annotation;
+    };
```

Comment:
This is a copy of `WorldPoses`, which is defined above. I had to make a copy of the definition instead of doing something like `using ChangedWorldPoses = WorldPoses` in order to avoid naming/scope collisions. Is this okay, or is there something else I can do to avoid this duplication? I can also add some documentation that explains why `WorldPoses` and `ChangeWorldPoses` have the same definition.

Reply:
I think it's okay. The types do have to be different (not aliases of each other). Adding a comment would suffice.

Reply:
Comment has been added in https://github.com/ignitionrobotics/ign-physics/pull/238/commits/1433ddbc73999fc149e25dc206448f4ee7b52ac2